### PR TITLE
Added JSON API support for `GetPGPLoginDetails`

### PR DESCRIPTION
**Pull request by Kumaravinash9**

### CODE REVIEW

**Comment by Kumaravinash9** [comment]
@csoler please review this PR.

File: libretroshare/src/retroshare/rsinit.h [hunk 1]
```diff
@@ -276,6 +276,10 @@ class RsAccounts
 	 * @return true on success, false otherwise
 	 */
 	static int GetPGPLogins(std::list<RsPgpId> &pgpIds);
+     /**
+	  * @jsonapi{development}
```
Comment (csoler) [inline libretroshare/src/retroshare/rsinit.h:280]: you need to add an entry for each parameter. See the other methods in rsinit.h

Comment (Kumaravinash9) [reply libretroshare/src/retroshare/rsinit.h:280]:
```/** @brief Lock profile directory
	 * param[in] accountDir account directory to lock
	 * param[out] lockFilePath path of the created lock-file
	 */
```
You mean to say entry like this i need to add

Comment (csoler) [reply libretroshare/src/retroshare/rsinit.h:280]: yes

**Comment by defnax** [comment]
this is done?

**Comment by defnax** [comment]
@csoler hope needed api stuff will be added soon to master to have it for retroshare mobile

**Pull request closed without merging**